pwa/src/templates/templateParts/databaseForm/DatabaseFormTemplate.tsx

```diff
@@ -0,0 +1,172 @@
+  };
+
+  React.useEffect(() => {
+    setIsLoading({ databaseForm: createOrEditDatabase.isLoading });
+  }, [createOrEditDatabase.isLoading]);
+
+  React.useEffect(() => {
+    database && handleSetFormValues(database);
+  }, [database]);
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} id={formId} className={styles.formContainer}>
+      <div className={styles.gridContainer}>
+        <div className={styles.grid}>
+          <FormField>
+            <FormFieldInput>
+              <FormFieldLabel>{t("Name")}</FormFieldLabel>
+              <InputText
+                {...{ register, errors }}
+                name="name"
+                validation={enrichValidation({ required: true })}
+                disabled={isLoading.databaseForm}
+                // ariaLabel={t("Enter name")}
+              />
+            </FormFieldInput>
+          </FormField>
+          <FormField>
+            <FormFieldInput>
+              <FormFieldLabel>{t("Reference")}</FormFieldLabel>
+              <InputText
+                {...{ register, errors }}
+                name="reference"
+                validation={enrichValidation({ required: true })}
+                disabled={isLoading.databaseForm}
+                // ariaLabel={t("Enter name")}
+              />
+            </FormFieldInput>
+          </FormField>
+
+          <FormField>
+            <FormFieldInput>
+              <FormFieldLabel>{t("Description")}</FormFieldLabel>
+              <Textarea
+                {...{ register, errors }}
+                name="description"
+                disabled={isLoading.databaseForm}
+                // ariaLabel={t("Enter description")}
+              />
+            </FormFieldInput>
+          </FormField>
+
+          <FormField>
+            <FormFieldInput>
+              <FormFieldLabel>{t("Version")}</FormFieldLabel>
+              <InputText
+                {...{ register, errors }}
+                name="version"
+                disabled={isLoading.databaseForm}
+                defaultValue={database?.version ?? "0.0.0"}
+                // ariaLabel={t("Enter version")}
+              />
+            </FormFieldInput>
+          </FormField>
+
+          <FormField>
+            <FormFieldInput>
+              <FormFieldLabel>{t("Select organizations")}</FormFieldLabel>
+
+              {getOrganizations.isLoading && <Skeleton height="50px" />}
+              {getOrganizations.isSuccess && (
+                <SelectMultiple
+                  options={getOrganizations.data.map((organization: any) => ({
+                    label: organization.name,
+                    value: organization.id,
+                  }))}
+                  name="organizations"
+                  {...{ register, errors, control }}
+                  disabled={isLoading.databaseForm}
+                  // ariaLabel={t("Select one or more organizations")}
+                />
```

Comment:
As discussed, I went to look for a solution to the bug I found and I have found the solution.
In order to stop adding organizations that are already added to a different database, we can just filter out the ones that are like so:
```
 .filter((organization: any) => !organization.database || organization.database.id === database.id)
```
Add this filter between `getOrganizations.data` and `.map((organization.....` and it will filter out all organizations that already have a database, but it will keep the ones that are connected to the database you are looking at